code/modules/mob/language/station_vr.dm

```diff
@@ -88,23 +88,26 @@
 	desc = "Shadekin seem to always know what the others are thinking. This is probably why."
 	speech_verb = "mars"
 	ask_verb = "mars"
-	exclaim_verb = "mars"
+	exclaim_verb = "MARS"
 	colour = "changeling"
 	key = "m"
 	machine_understands = FALSE
 	flags = WHITELISTED | HIVEMIND
 
-/datum/language/human/slavic
+/datum/language/slavic
```

Comment:
This seems to be listed right after Shadekin Empathy language on a character's Player Panel. 
![image](https://user-images.githubusercontent.com/38586851/76151342-f7a42980-6081-11ea-856c-ed9bbf7f80e6.png) Any way to have it be at a more appropriate place, like, where the languages aren't event char or robotic. Just a purrsonal preference regarding where it's placed.

Reply:
Won't fix.

Reply:
Alrighty, then.